cmd/multitool/multitool.go

```diff
@@ -510,5 +510,6 @@ func verify(args []string) {
 }
 
 func zerohash(_ []string) {
-	fmt.Println(bc.Hash{}.String())
+	var h bc.Hash
+	fmt.Println(h.String())
```

Comment:
No: the protobuf `String` method takes a pointer receiver and so needs an addressable object.